Framework/Geometry/test/BeamlineRayTracerTest.h

```diff
@@ -0,0 +1,615 @@
+#ifndef BEAMLINERAYTRACERTEST_H_
+#define BEAMLINERAYTRACERTEST_H_
+
+#include "MantidGeometry/Instrument/ComponentInfo.h"
+#include "MantidGeometry/Instrument/DetectorInfo.h"
+#include "MantidGeometry/Instrument/InstrumentDefinitionParser.h"
+#include "MantidGeometry/Instrument/InstrumentVisitor.h"
+#include "MantidGeometry/Instrument/RectangularDetector.h"
+#include "MantidGeometry/Objects/BeamlineRayTracer.h"
+#include "MantidGeometry/Objects/InstrumentRayTracer.h"
+#include "MantidKernel/ConfigService.h"
+#include "MantidKernel/ProgressText.h"
+#include "MantidKernel/Strings.h"
+#include "MantidTestHelpers/ComponentCreationHelper.h"
+
+#include <boost/make_shared.hpp>
+#include <cxxtest/TestSuite.h>
+
+using namespace Mantid::Geometry;
+using Mantid::Kernel::ConfigService;
+using Mantid::Kernel::ProgressText;
+using Mantid::Kernel::V3D;
+using namespace Mantid::Kernel::Strings;
+using namespace ComponentCreationHelper;
+namespace RayTracer = Mantid::Geometry::BeamlineRayTracer;
+
+class BeamlineRayTracerTest : public CxxTest::TestSuite {
+public:
+  // This pair of boilerplate methods prevent the suite being created statically
+  // This means the constructor isn't called when running other tests
+  static BeamlineRayTracerTest *createSuite() {
+    return new BeamlineRayTracerTest();
+  }
+  static void destroySuite(BeamlineRayTracerTest *suite) { delete suite; }
+
+  BeamlineRayTracerTest() {
+    // Start logging framework
+    Mantid::Kernel::ConfigService::Instance();
```

Comment:
I don't think this is necessary? If not remove "ConfigService" include and `using` too.